content/doc/developer/plugin-development/choosing-jenkins-baseline.adoc

```diff
@@ -32,6 +32,9 @@ There are link:https://stats.jenkins.io/pluginversions/[statistics of core versi
 These are updated monthly.
 When updating the core dependency, choose the link:/changelog-stable/[newest LTS version] that doesn't exclude a majority of your existing users (by requiring a newer Jenkins than they have).
 
+NOTE:  if you are packaging a pure API library (one that does not depend on Jenkins APIs) then you should ignore newer jenkins versions and pick an older LTS.
+Something around 1 year old that does not have too many detached plugins makes a good choice and *PLACEHOLDER_OLDEST_LTS* would be a reasonable candidate.
```

Comment:
Ideally the plugin manifest could contain an option which could prevent the older baseline from even considering detached plugins for these types of plugins, but that is for another day.

Reply:
https://issues.jenkins.io/browse/JENKINS-28942